src/components/pages/Offers/Offer/Stocks/__specs__/Stocks.spec.jsx

```diff
@@ -980,6 +980,28 @@ describe('stocks page', () => {
               expect(pcapi.bulkCreateOrEditStock).not.toHaveBeenCalled()
             })
 
+            it('should be able to edit stock when remaining quantity is unlimited and there is existing bookings', async () => {
+              const eventStock = {
+                ...defaultStock,
+                beginningDatetime: '2020-12-20T22:00:00Z',
+                quantity: null,
```

Comment:
```suggestion
                quantity: 'unlimited,
```

Sinon ça ne correspond pas avec le nom du test non ?

Reply:
C'est quand elle est null qu'elle est considérée illimitée